docs/examples/cluster.yaml

```diff
@@ -1,25 +1,78 @@
-# https://docs.aws.amazon.com/emr/latest/APIReference/API_RunJobFlow.html
-Name: mycluster
-JobFlowRole: myjobflowrole
-ServiceRole: myservicerole
-ReleaseLabel: emr-5.24.1
+---
 Applications:
-- Name: HBase
-- Name: Hive
-- Name: Spark
+- Name: "Spark"
+- Name: "Hive"
+- Name: "HBase"
+Configurations:
```

Comment:
In order to try and keep the YAML files to a reasonable size, could you move this block out to a `configurations.yaml` file, and add code in `emr_launcher.py` to read from that if it's found.

Reply:
Fixed by https://github.com/dwp/emr-launcher/pull/2/commits/3f0a7c95c4545e4fcdccbf8c85cf733250be4623